src/app/components/DownloadItem/__tests__/DownloadItem.test.js

```diff
@@ -552,6 +552,61 @@ describe('DownloadItem component', () => {
     })
   })
 
+  describe('when a download is in "interrupted" state', () => {
+    test('displays the "Manually Download" button', async () => {
+      const mockCallback = jest.fn()
+      setup({
+        actionsList: [
+          [
+            {
+              label: 'Manually Download',
+              isActive: true,
+              isPrimary: false,
+              callback: mockCallback
+            }
+          ]
+        ],
+        percent: 0,
+        state: downloadStates.interrupted,
+        downloadLinks: { 'download-id': ['https://example.com/file1.zip'] }
+      })
+
+      global.open = jest.fn()
```

Comment:
Isn't this 
```suggestion
     global.window.open = jest.fn()
```
like it is in `src/app/components/Layout/__tests__/Layout.test.js`? I'm surprised the mock works this way

Reply:
These do the same exact thing actually. `global.window.open` is just more verbose, but I can change it to that to be consistent if that's the syntax we used in a different file.

Reply:
Lets just call it consistently here like the other one